### ISSUE TITLE
no assets in "build" folder

### ISSUE THREAD

**Issue by royts** [timroes/lvlup-game#27]
Hi,
I cloned the project and run `npm install`.
Then I run `npm start`.
I can see the login page and then the "game" page, but it looks broken.

Looking in the console I see that it can't load some files: `assets/fontello/css/fontello.css , assets/loading.svg` , etc.

When I look into the "build" folder indeed - the files aren't there.

Maybe something ir broken with gulp tasks?

---

BTW - tried to install and run with many gulp versions and node versions.

**Comment by timroes** [comment]
npm install should have run a gulp build. Could you Paste the output of that?

Also could you try running gulp build manually and check if that works?

**Comment by royts** [comment]
I tried running `gulp build` manually, same result.
Here is the output:

```
[10:45:39] Requiring external module babel-core/register
[10:45:39] Using gulpfile ~/dev/lvlup-game/gulpfile.babel.js
[10:45:39] Starting 'admin:fontello'...
[10:45:39] Starting 'admin:scripts'...
[10:45:39] Starting 'admin:styles'...
[10:45:39] Starting 'admin:html'...
[10:45:39] Starting 'admin:libs'...
[10:45:39] Starting 'admin:libs:styles'...
[10:45:39] Starting 'admin:templates'...
[10:45:39] Starting 'client:fontello'...
[10:45:39] Starting 'client:scripts'...
[10:45:39] Starting 'client:styles'...
[10:45:39] Starting 'client:html'...
[10:45:39] Starting 'client:libs'...
[10:45:39] Starting 'client:libs:styles'...
[10:45:39] Starting 'client:templates'...
[10:45:39] Starting 'screen:fontello'...
[10:45:39] Starting 'screen:scripts'...
[10:45:39] Starting 'screen:styles'...
[10:45:39] Starting 'screen:html'...
[10:45:39] Starting 'screen:libs'...
[10:45:39] Starting 'screen:libs:styles'...
[10:45:39] Starting 'screen:templates'...
[10:45:39] Starting 'shared:fontello'...
[10:45:39] Starting 'shared:scripts'...
[10:45:39] Starting 'shared:styles'...
[10:45:39] Starting 'shared:html'...
[10:45:39] Starting 'shared:libs'...
[10:45:39] Starting 'shared:libs:styles'...
[10:45:39] Starting 'shared:templates'...
[10:45:39] Starting 'server:build'...
[10:45:39] Finished 'admin:templates' after 78 ms
[10:45:39] Finished 'screen:fontello' after 60 ms
[10:45:39] Starting 'screen:assets'...
[10:45:39] Finished 'screen:templates' after 53 ms
[10:45:39] Finished 'shared:fontello' after 51 ms
[10:45:39] Starting 'shared:assets'...
[10:45:39] Finished 'shared:html' after 48 ms
[10:45:39] Finished 'screen:assets' after 12 ms
[10:45:39] Finished 'shared:assets' after 9.55 ms
[10:45:39] Finished 'screen:libs:styles' after 245 ms
[10:45:39] Finished 'client:libs' after 261 ms
[10:45:39] Finished 'admin:libs' after 286 ms
[10:45:39] Finished 'screen:libs' after 251 ms
[10:45:40] Finished 'shared:scripts' after 348 ms
[10:45:40] Finished 'shared:templates' after 579 ms
[10:45:40] Finished 'shared:styles' after 584 ms
[10:45:40] Finished 'screen:styles' after 597 ms
[10:45:40] Finished 'screen:scripts' after 624 ms
[10:45:41] Finished 'screen:html' after 1.39 s
[10:45:41] Starting 'screen:build'...
[10:45:41] Finished 'screen:build' after 15 μs
[10:45:41] Finished 'admin:scripts' after 1.77 s
[10:45:41] Finished 'admin:styles' after 1.66 s
[10:45:41] Finished 'client:html' after 1.4 s
[10:45:41] Finished 'client:scripts' after 1.44 s
[10:45:41] Finished 'client:styles' after 1.51 s
[10:45:41] Finished 'client:templates' after 1.51 s
[10:45:41] Finished 'admin:fontello' after 1.93 s
[10:45:41] Starting 'admin:assets'...
[10:45:41] Finished 'admin:assets' after 1.82 ms
[10:45:41] Finished 'admin:html' after 1.58 s
[10:45:41] Finished 'server:build' after 1.5 s
[10:45:45] Finished 'client:fontello' after 5.41 s
[10:45:45] Starting 'client:assets'...
[10:45:45] Finished 'client:assets' after 1.6 ms
```

**Comment by royts** [comment]
@timroes , if it is working on your side, can you please post here:
- node version
- gulp (global) version
- cairo version
- pkg-config version

thanks!

**Comment by timroes** [comment]
Fixed in f0beb4321cd2ada889620c3ea778ef3de8c8d004

That actually was a bug in the buildsystem. You should be able to build now properly.

**Comment by royts** [comment]
works great now.
Thank you!
